deprecated/deprecated-functions.php

```diff
@@ -63,7 +64,11 @@ function dokan_depricated_widget_classes() {
 add_action( 'woocommerce_before_main_content', 'dokan_depricated_widget_classes' );
 
 /**
- * Depricated function for render seller metabox in product
+ * Deprecated function for render seller metabox in product
+ *
+ * @since 3.0.0
+ *
+ * @deprecated 3.0.0 Use \WeDevs\Dokan\Admin\Hooks::seller_meta_box_content instead of this function
```

Comment:
It is being used in https://github.com/getdokan/dokan-pro/blob/059283a2b65bf32b47620432228e851a143a2f78/modules/single-product-multiple-vendor/includes/products-admin.php#L44